packages/makeswift/package.json

```diff
@@ -31,8 +31,9 @@
     "@babel/generator": "^7.18.13",
     "@babel/template": "^7.18.10",
     "@babel/traverse": "^7.18.13",
+    "chalk": "^4.1.2",
     "commander": "^9.4.0",
-    "create-next-app": "^12.2.5",
+    "create-next-app": "12.x",
```

Comment:
`^12.3.1` was fine, it means "12.x" already

Reply:
done